src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -3,6 +3,8 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import static edu.kis.vh.nursery.stack.IntStackInterface.EMPTY_STACK_INDEX;
+
 public class RhymersJUnitTest {
```

Comment:
About testIsFull: Is it a correct approach to fixing errors showed by tests? 
Errors are still within the project. If STACK_CAPACITY is 12 then 12 elements should be added to the stack and it should be FULL.